contracts/ronin/gateway/RoninGatewayV2.sol

```diff
@@ -538,24 +520,16 @@ contract RoninGatewayV2 is
   function _getVoteWeight(
     IsolatedGovernance.Vote storage _v,
     bytes32 _hash
-  ) internal view returns (uint256 _totalWeight, uint256 _trustedWeight) {
-    address[] memory _consensusList = IRoninValidatorSet(getContract(ContractType.VALIDATOR)).getValidators();
-    uint256[] memory _trustedWeights = IRoninTrustedOrganization(getContract(ContractType.RONIN_TRUSTED_ORGANIZATION))
-      .getConsensusWeights(_consensusList);
-    // TODO: uncomment below logic
-
-    // unchecked {
-    //   for (uint _i; _i < _bridgeOperators.length; ++_i) {
-    //     if (
-    //       _flags[_i].hasFlag(EnumFlags.ValidatorFlag.BridgeOperator) && _v.voteHashOf[_bridgeOperators[_i]] == _hash
-    //     ) {
-    //       _totalWeight++;
-    //       if (_trustedWeights[_i] > 0) {
-    //         _trustedWeight++;
-    //       }
-    //     }
-    //   }
-    // }
+  ) internal view returns (uint256 _totalWeight) {
+    address[] memory _bridgeOperators = IBridgeManager(getContract(ContractType.BRIDGE_MANAGER)).getBridgeOperators();
+
+    unchecked {
+      for (uint _i; _i < _bridgeOperators.length; ++_i) {
+        if (_v.voteHashOf[_bridgeOperators[_i]] == _hash) {
+          _totalWeight++;
```

Comment:
Since we have the weight for each bridge operator, why don't we use it instead of counting like this?

Reply:
make sense.

Reply:
what if `_v.voteHashOf[_bridgeOperators[_i]] != _hash`